.github/workflows/arm-incremental-typespec-preview.yaml

```diff
@@ -1,6 +1,14 @@
 name: ARM Incremental TypeSpec (Preview)
 
-on: pull_request
+on:
+  pull_request:
+    paths:
+      # Always trigger on changes to WF itself
+      - ".github/**"
```

Comment:
Won't that case it to trigger on any workflow updates? Did you intend to scope to itself or did you want to also catch the JS file updates?

Reply:
> Won't that case it to trigger on any workflow updates?

Yes, but this is just an optimization to skip unnecessary runs.  If the check does trigger, it will return false, which is effectively the same as being skipped.

> Did you intend to scope to itself or did you want to also catch the JS file updates?

My intention was to catch any change that could impact this workflow (including common JS files), and I don't mind if it sometimes runs unnecessarily.